docs_src/microservices/device/supported/device-usb-camera/walkthrough/deployment.md

```diff
@@ -109,6 +109,74 @@ Follow this guide to deploy and run the service.
     !!! Note 
         The `jq -r` option is used to reduce the size of the displayed response. The entire device with all information can be seen by removing `-r '"deviceName: " + '.devices[].name'', and replacing it with '.'`
 
+1. Add credentials to for the rtsp stream.
+
+    !!! note
+        If you want to disable rtsp authentication entirely, you must [build a custom image](../walkthrough/custom-build.md).
+
+    === "Scripts"
+
+        !!! note 
+            If running in secure mode the [JWT token](../../../../../security/Ch-APIGateway.md#creating-access-token-for-api-gateway-authentication) generated previously is needed for mapping credentials.
+
+        a. Enter JWT token if in secure mode.  
+            ![](../images/enter-jwt-token.png)  
+        b. Set the username.  
+            ![](../images/set-username.png)  
+        c. Set the password.  
+            ![](../images/set-username.png)  
+
+        Successful:
+        ```bash
+        Dependencies Check: Success
+            Consul Check: ...
+                            [
+        {
+            "Resource": "key",
+            "Access": "read",
+            "Allow": false
+        },
+        {
+            "Resource": "key",
+            "Access": "write",
+            "Allow": false
+        }
+        ]
+
+        curl --data '{
+            "apiVersion":"v3",
+            "secretName": "rtspauth",
+            "secretData":[
+                {
+                    "key":"username",
+                    "value":"username"
+                },
+                {
+                    "key":"<redacted>",
+                    "value":"<redacted>"
+                }
+            ]
+        }' -H Authorization:Bearer <redacted> -X POST http://localhost:59983/api/v3/secret
+        Response [201] {
+        "apiVersion": "v3",
+        "statusCode": 201
+        }
+
+
+        Success
+        ```
+        
+    === "Consul UI"
+        !!! note 
+                If running in secure mode the [Consul ACL](../../../../../security/Ch-Secure-Consul.md#how-to-get-consul-acl-token) is needed to access consul.
```

Comment:
The consul-acl token is needed to access the consul ui.  Also, I'm tempted to remove the "generated previously" part of the sentence because right now the usb docs don't have any info on generating that stuff. That's why I include the link in the sentence.

Reply:
There is a bit of an inconsistency here though. In secure mode, there is no concept of Insecure Secrets, so this method only works in non-secure mode.